src/Backend/Core/Engine/Header.php

```diff
@@ -361,7 +361,7 @@ public function parseJS()
         if (Authentication::getUser()->isAuthenticated()) {
             $interfaceLanguage = (string) Authentication::getUser()->getSetting('interface_language');
         } else {
-            $interfaceLanguage = Language::getInterfaceLanguage();
+            $interfaceLanguage = BL::getInterfaceLanguage();
```

Comment:
Can you use BackendLanguage instead?
So it is consistent with your other changes.